wordpress-mobile/WordPress-Android
# Adds: Missing database schema for version 9

Fixes #https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/2579

Dependant fluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2580

To test: 
- There is nothing to test
- Make sure that the CI passes successfully in this PR as well as [Dependant fluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2580)


⚠️  Merge Instructions 
- [x] Make sure the [Dependant fluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2580) is merged 
- [x] Update the FluxC PR 
- [x] Remove Not Ready for Merge label
- [x] Merge as normal 

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
